torch/csrc/jit/codegen/cuda/scheduler/reduction_heuristic.h

```diff
@@ -21,14 +23,19 @@ struct ReductionParams {
   // Perform multiple reductions per block?
   bool multiple_reds_per_blk = false;
   // Unrolling factor
-  int64_t loop_unroll = 4;
+  int64_t loop_unroll = 1;
```

Comment:
Is the `loop_unroll` factor always 1 now? If so, do we need to split by `loop_unroll` then?

Reply:
Thanks, it won't be for long.